app/AbstractRequest.php

```diff
@@ -341,7 +344,38 @@ protected function responseFromCache(): ?Response
         }
 
         $fromCache = Cache::tags($this->cacheTags)->get($this->cacheKey());
-        return $fromCache ? new Response(...$fromCache) : null;
+        if ($fromCache) {
+            $this->sentLogs = $fromCache['logs'];
+            return new Response(...$fromCache['response']);
+        }
+        return null;
+    }
+
+    /**
+     * Return cache key string based on this class, URL, and request body
+     * @return string
+     */
+    public function cacheKey(): string
+    {
+        return hash(
+            'sha256',
+            json_encode([
+                self::class,
+                $this->getURL(),
+                $this->encodeBody(),
+                config('api-request.cache_key_seed', 'v2024.8.6'),
```

Comment:
This is what we designed the seed for, the shape of the data we cached is changing, so we're altering the seed, which will cause misses on all previously cached data. We'll want to give this a minor version bump and note it in the release notes.